CHANGELOG.rst

```diff
@@ -27,6 +27,17 @@ API Changes
   shiftOrigin is equivalent to withOrigin, which still exists as a valid
   name for this action. (#1085)
 - Deprecated galsim.wfirst module.  Now called galsim.roman. (#1088)
+- Changed the default ii_pad_factor for PhaseScreenPSF and OpticalPSF to 1.5.
+  The old default of 4.0 (matching the InterpolatedImage default) is not
+  necessary for objects that will not be significantly sheared, which
+  PSF objects typically aren't.  For almost all use cases, this will be
+  simply a performance improvement, but if you need the higher ii_pad_factor
+  for some reason, you will now need to manually specify it. (#1089)
```

Comment:
Could be I missed it, but I didn't see in the tests that you had any where you had to manually set a higher value.  Does that mean that all of our tests passed with this lower default?  Interesting...

Reply:
Yes.  I'm pretty sure if you're not shearing an interpolated image, then the pad factor doesn't need to be set very high.  Certainly, dropping to 1.5 for all the phase_psf cases didn't cause any problems, probably because we never shear these objects.  When it's used for a PSF, it would be weird to shear it.  (The shape usually comes from aberrations, not an applied shear.) 

The pad factor is important though if you're going to shear the object, like we often do with RealGalaxy for instance, so the default of 4 is preserved in that case, as well as the generic InterpolatedImage class, where we don't know what it will be used for.

Reply:
Makes sense, thanks.